webapp/src/types.ts

```diff
@@ -18,3 +18,10 @@ export type GlobalState = ReduxGlobalState & {
 }
 
 export type InputFieldType = HTMLInputElement | HTMLTextAreaElement;
+
+export type RadioOptionsType = {
+    value: string;
+    checked: boolean;
+    id: string;
+    message: string;
```

Comment:
```suggestion
    label: string | ReactNode;
```

Reply:
You have not done this change, yet you resolved this comment. Any reason? @ayusht2810

Reply:
I thought we need to do this on individual props only. updated now.